packages/flutter_blue_plus_platform_interface/lib/src/bluetooth_msgs.dart

```diff
@@ -633,15 +656,16 @@ class BmSetNotifyValueRequest {
   final DeviceIdentifier remoteId;
   final Guid serviceUuid;
   final Guid characteristicUuid;
+  final int characteristicIndex;
   final Guid? primaryServiceUuid;
   final bool forceIndications;
   final bool enable;
 
-
   BmSetNotifyValueRequest({
     required this.remoteId,
     required this.serviceUuid,
     required this.characteristicUuid,
+    required this.characteristicIndex,
```

Comment:
Should we default this to `0` and make it optional in order to avoid a breaking change?

```suggestion
    this.characteristicIndex = 0,
```

https://github.com/chipweinberger/flutter_blue_plus/issues/795#issuecomment-2461566048